pkg/netpol/eval/resources.go

```diff
@@ -203,6 +207,54 @@ func (pe *PolicyEngine) upsertNamespace(ns *corev1.Namespace) error {
 	return nil
 }
 
+// checkLegalOwnerRefNameToPodLabels returns error if there are pod resources with same ownerReferences name but different labels
+func (pe *PolicyEngine) checkLegalOwnerRefNameToPodLabels(newPod *k8s.Pod) error {
```

Comment:
rename function to be more specific/relevant, such as `checkConsistentLabelsForPodsOfSameOwner`